src/apps/loan-list/materials/utils/status-badge.tsx

```diff
@@ -26,29 +29,61 @@ const StatusBadge: FC<StatusBadgeProps> = ({
 
   if (daysBetweenTodayAndDue < threshold.danger && dangerText) {
     return (
-      <div className="status-label status-label--danger">{dangerText}</div>
+      <div
+        className={clsx("status-label status-label--danger", [
+          { "hide-on-desktop ml-16": mobileVersion }
+        ])}
+      >
+        {dangerText}
+      </div>
```

Comment:
Since this block is repeated multiple times with `status-label--*` as the only part changing I think you should either make a function or a component (I think I perefer a component, eg. StatusBadgeText) for that part of the rendering:
```typescript
<div
        className={clsx(`status-label status-label--${type}`, [
          { "hide-on-desktop ml-16": mobileVersion }
        ])}
      >
        {text}
      </div>
```